whois-commons/src/main/java/net/ripe/db/whois/common/apiKey/OAuthSession.java

```diff
@@ -87,19 +87,33 @@ public static class ScopeFormatter {
 
         final String appName;
         final String scopeType;
+        final String scopeEnv;
         final String scopeKey;
 
         public ScopeFormatter(final String scope) {
-            final String[] parts = scope.split(":|\\.");
+            final String[] parts = scope.split("[:.]");
 
-            if(parts.length == 0 || parts.length < 2) {
+            if (parts.length <= 4){
+                this.appName = parts[0];
```

Comment:
I think we should use the seperate environment scope rather than having in environemnt in middle of one scope. This makes it very clear which scope is for environment and which scope is for mntnr

Reply:
yes, done